distributed/scheduler.py

```diff
@@ -2747,7 +2747,11 @@ def transition_processing_released(self, key):
                 self.total_occupancy -= duration
                 self.check_idle_saturated(w)
                 self.release_resources(key, w)
-                self.worker_comms[w].send({'op': 'release-task', 'key': key})
+                try:
+                    self.worker_comms[w].send({'op': 'release-task',
+                                               'key': key})
+                except EnvironmentError:
```

Comment:
It's specifically a CommClosedError, right?

Reply:
Replaced with `CommClosedError`